rocketpy/environment/environment.py

```diff
@@ -3126,7 +3126,18 @@ def all_plot_info_returned(self):
         ------
         plot_info : Dict
             Dict of data relevant to plot externally
+
+        Warning
+        -------
+        Deprecated in favor of `utilities.get_instance_attributes`.
+
         """
+        warnings.warn(
+            "The method 'all_plot_info_returned' is deprecated. "
```

Comment:
Deprecation message should contain the specific version in which the method was deprecated, and also the version in which that will be completely removed. Also, it should indicates new methods to be used in the place of the deprecated method.

My suggestion is to deprecate it in v1.2.0 and fully remove it in v.1.4.0

warnings.warn(
        "old_method() is deprecated as of version X.Y and will be removed in version X.(Y+2). "
        "Please use new_method() instead.", 
        DeprecationWarning, 
        )